scipy/special/_logsumexp.py

```diff
@@ -44,7 +44,8 @@ def logsumexp(a, axis=None, b=None, keepdims=False, return_sign=False):
     sgn : ndarray
         If return_sign is True, this will be an array of floating-point
         numbers matching res and +1, 0, or -1 depending on the sign
```

Comment:
While we're at it, isn't this technically about the exponential of the sign of the result, assuming "result" is what the function returns?

Reply:
Done